rtrlib/rtr/packets.c

```diff
@@ -342,7 +342,6 @@ static bool rtr_pdu_check_size (const struct pdu_header *pdu) {
     min_size = 4 + sizeof(struct pdu_error);
     if (err_pdu->len < min_size) {
       RTR_DBG1("PDU it to small to contain \"Length of Error Text\" field!");
```

Comment:
Can you also fix the typo `s/it/is/` and `s/to small/too small/` (latter also applies for the subsequent cases).